app/controllers/users_controller.rb

```diff
@@ -11,22 +10,68 @@ def index
     @users = User.all
   end
 
+  def new
+    if current_user.admin?
+      @user = User.new
+    else
+      set_flash_message :alert, :must_be_admin
+      redirect_to current_user
+    end
+  end
+
   def create
     @user = User.new(user_params)
-
     if @user.save
-      flash[:success] = "\"#{@user.name}\" added -- Welcome to the Motley Tones"
-      redirect_to user_path(@user.id)
+      total = current_user.class.all.count
+      flash[:notice] = "New pirate added.  There #{(total == 1) ? "is" : "are"} now #{total} on the roster"
```

Comment:
There may also be a way to leverage Rails' language helpers (think like `pluralize`) to control the 'is'/'are' distinction above.